src/dBFT/DBFTPlugin.cs

```diff
@@ -0,0 +1,54 @@
+using Akka.Actor;
+using Neo.ConsoleService;
+using Neo.Network.P2P;
+using Neo.Plugins;
+using Neo.Wallets;
+
+namespace Neo.Consensus
+{
+    public class DBFTPlugin : Plugin, IConsensusProvider, IP2PPlugin
+    {
+        private IWalletProvider walletProvider;
+        private IActorRef consensus;
+        private bool started = false;
```

Comment:
```suggestion
        public override string Name => "dBFT";
        public override string Description => "Enable consensus";
        private IWalletProvider walletProvider;
        private IActorRef consensus;
        private bool started = false;
```
![image](https://user-images.githubusercontent.com/38493437/104557892-f027b680-567c-11eb-8b9c-5b130594a443.png)

Reply:
I think we don't need to change the name.

Reply:
The description has been added.